extensions/amp-story/1.0/amp-story.js

```diff
@@ -1177,6 +1200,7 @@ export class AmpStory extends AMP.BaseElement {
             pageIndex,
             this.getPageCount(),
             targetPage.element.id,
+            oldPage.element.id,
```

Comment:
Can `oldPage` be undefined and trigger an error, eg: on first navigation?

Reply:
Thanks for catching this, I went ahead and moved the code to avoid this problem.